status/flowcells.py

```diff
@@ -26,7 +26,14 @@ def list_flowcells(self):
         xfc_view = self.application.x_flowcells_db.view("info/summary",
                                                      descending=True)
         for row in xfc_view:
-            temp_flowcells[row.key] = row.value
+            if len(row.value['startdate']) < 7:
+                row.value['startdate'] = "20{}-{}-{}".format(row.value['startdate'][0:2],row.value['startdate'][2:4],row.value['startdate'][4:6]) 
+                temp_flowcells[row.key] = row.value
+            else:
+                #Novaseq date format fix
```

Comment:
Would be nice to paste an example of what this looks like in full

Reply:
Good point!